src/styles/atoms/input.scss

```diff
@@ -22,3 +23,7 @@ select {
     @apply border-primary;
```

Comment:
The submit button suddenly has a gray border, which was not there before:

![image](https://user-images.githubusercontent.com/88790097/142890450-c1406154-a45e-4924-a32c-76787d51f36d.png)

Reply:
removed the border